packages/zoe/src/zoe.js

```diff
@@ -439,7 +422,34 @@ const makeZoe = (additionalEndowments = {}) => {
         return completeOffers(instanceHandle, offerHandles);
       },
 
-      makeInvitation,
+      // Make a Zoe invite payment with an extent that is a mix of credible
+      // information from Zoe (the `handle` and `instanceHandle`) and
+      // other information defined by the smart contract (the mandatory
+      // `inviteDesc` and the optional`options.customProperties`).
+      // Note that the
+      // smart contract cannot override or change the values of `handle`
+      // and `instanceHandle`.
```

Comment:
github lost the correspondence with the original lines, so there's no value in retaining the original line breaks. 

fix it or don't as you prefer.

Reply:
Done.